examples/messages.vroom

```diff
@@ -7,10 +7,16 @@ the command that causes them if you want it to work.
 This mechanism allows vroom to know what lines errors occur on. You can match
 multiple messages on one command.
 
+  :set cmdheight=99
+
   :echomsg "First" | echomsg "Second"
   ~ First
   ~ Second
 
+Note sometimes multiple lines of output cause vroom to get stuck at a "Hit ENTER
```

Comment:
Might be useful to include the word "Neovim" here somewhere, since AIUI the only issue is with Neovim specifically?

Reply:
Done.